lib/util/ts_parser.js

```diff
@@ -497,3 +591,18 @@ shaka.util.TsParser.PMT;
  */
 shaka.util.TsParser.PES;
 
+
+/**
+ * @typedef {{
+ *   data: !Uint8Array,
+ *   type: string,
+ *   time: ?number
+ * }}
+ *
+ * @summary PES.
+ * @property {!Uint8Array} data
+ * @property {string} type
```

Comment:
You're assigning ints to this, but it's defined as a string.

Also, the caller of the parser never uses this field, so maybe you could just delete it.

Reply:
This is used in `ts_cea_parser.j`s to be able to identify if it is a NALU of SEI type

Reply:
Ah, indeed.  I overlooked that.  In any case, you're assigning numbers to it, but you have it typed as string.

Reply:
done!